handlers/file-get.js

```diff
@@ -2,11 +2,49 @@
  * file-get
  * our Request handler.
  */
-
+const path = require("path");
 const ABBootstrap = require("../AppBuilder/ABBootstrap");
 // {ABBootstrap}
 // responsible for initializing and returning an {ABFactory} that will work
 // with the current tenant for the incoming request.
+const pathUtils = require("../utils/pathUtils.js");
+
+const getFilePath = async (req, file, needOriginalImage) => {
+   let filePath = file.pathFile;
+   switch (file.type) {
+      case "image/heic":
+      case "image/jpeg":
+      case "image/png":
+      case "image/tiff":
+      case "image/webp":
+         const parsedImagePath = path.parse(filePath);
+         const imageWEBPPath = path.join(
+            parsedImagePath.dir,
+            `${parsedImagePath.name}.webp`
+         );
+         if ((await pathUtils.checkPath(imageWEBPPath))) {
+            if (!needOriginalImage) filePath = imageWEBPPath;
+            break;
+         }
+         if (await pathUtils.checkPath(filePath))
+            req.worker(
```

Comment:
I think we should also convert after a file upload.

Reply:
https://github.com/digi-serve/ab_service_file_processor/pull/52/commits/a68698dfafb5fd42c982db2535efaf36eef2c54a
https://github.com/digi-serve/ab_service_api_sails/pull/120